libpod/events/config.go

```diff
@@ -50,6 +50,10 @@ type Event struct {
 type Details struct {
 	// ID is the event ID
 	ID string
+	// ContainerCreateInspectData includes the payload of the container's
+	// inspect data. Only set when events_container_create_inspect_data is
+	// set true in containers.conf.
+	ContainerCreateInspectData string
```

Comment:
can you add omitempty here to make sure it does not show up in the API unless needed.

Reply:
Thanks for catching that! I removed it during manual testing but forgot putting it back.